packages/combobox/package.json

```diff
@@ -43,11 +43,12 @@
         "test": "echo \"Error: run tests from mono-repo root.\" && exit 1"
     },
     "files": [
-        "*.d.ts",
-        "*.js",
-        "*.js.map",
-        "/src/",
-        "custom-elements.json"
+        "**/*.d.ts",
```

Comment:
I think these are auto generated. Curious to know why the path relativity is changed here!

Reply:
Generated, yes. But, generates differently over time. The combobox project was open for multiple years before it landed during which the associated package generation was changed quite heavily.